### PR TITLE
Plugin Base updated to 3.1.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by abhishek-engg-mobilesdk**
Plugin Base & Android Native Sdk Update

### CODE REVIEW

File: core/CHANGELOG.md [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
+### Next Release
 
 ### 5.1.0
 - Android
   - Support for Android 13 notification permission.
   - Android Gradle Plugin version updated to `7.3.1`
   - Gradle version updated to `7.4`
+  - Compile SDK Version - 31
+  - Target SDK version - 31
+  - Support for Android SDK version `12.4.00-SNAPSHOT`
```
Comment (umangmoe) [inline core/CHANGELOG.md:12]: revert these versions from the changelog

File: inbox/CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,9 @@
     - Gradle version updated to `7.4`
     - Compile SDK Version - 31
     - Target SDK version - 31
+    - Support for Android SDK version `12.4.00-SNAPSHOT`
```
Comment (umangmoe) [inline inbox/CHANGELOG.md:11]: revert versions from here

File: core/CHANGELOG.md (hunk 1 above)
Comment (abhishek-engg-mobilesdk) [reply core/CHANGELOG.md]: Removed

File: inbox/CHANGELOG.md (hunk 2 above)
Comment (abhishek-engg-mobilesdk) [reply inbox/CHANGELOG.md]: Removed

**Pull request merged**